### PR TITLE
PYIC-3117 Added postMitigations lamdba with test. 

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
Add postMitigations stub lamdba. 

### What changed
Added postMitigations lamdba with test. 

### Issue tracking

- [PYI-3117](https://govukverify.atlassian.net/browse/PYIC-3117)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: di-ipv-cimit-stub/deploy/template.yaml [hunk 1]
```diff
@@ -49,13 +49,48 @@ Resources:
...
+  PostMitigationsFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_109: this requires a broad set of permissions
+    DependsOn:
+      - "PostMitigationsFunctionLogGroup"
+    Properties:
+      FunctionName: !Sub "postMitigations-${Environment}"
+      CodeUri: "../lambdas/post-mitigations"
+      Handler: uk.gov.di.ipv.core.postmitigations.PostMitigationsHandler::handleRequest
+      Runtime: java17
```
Comment (TalatAydinGDS) [inline di-ipv-cimit-stub/deploy/template.yaml]: could see java 11 in the build.gradle but here java 17.

**Review by TalatAydinGDS — CHANGES_REQUESTED** [review]

File: di-ipv-cimit-stub/deploy/template.yaml (hunk 1 above)
Comment (therealslimsladey) [reply di-ipv-cimit-stub/deploy/template.yaml]: Java 17 Lambdas still kick out some warnings so safer to stick with Java 11 for now, methinks?

Comment (TalatAydinGDS) [reply di-ipv-cimit-stub/deploy/template.yaml]: I think, should be use to the same version java11 for Lambda

Comment (rpayal) [reply di-ipv-cimit-stub/deploy/template.yaml]: Done.

**Pull request closed without merging**